packages/components/src/List/Toolbar/ColumnChooserButton/ColumnChooser/ColumnChooser.scss

```diff
@@ -0,0 +1,84 @@
+$tc-popover-border-color: rgb(226, 226, 226) !default;
+$tc-popover-border-width: 0.1rem !default;
+$tc-border-color: #ededed !default;
+$tc-icon-size: 1.2rem !default;
+$tc-height: 4rem;
+
+.tc-column-chooser {
+	:global(.tc-tooltip-body) {
+		margin: 0;
+	}
+
+	:global(.tc-tooltip-footer.tc-column-chooser-footer) {
+		justify-content: flex-end;
+	}
+
+	&-aria-hidden {
+		position: absolute;
+		left: -200em;
+		top: -20em;
+	}
+
+	&-header {
```

Comment:
For header/body/footer, it's safer to create a dedicated scss file, import them as css modules, and pass them to the tooltip header/body/footer elements.
If the classnames change in the tooltip parts, it won't impact the column chooser